app/plugin_key_value_store.go

```diff
@@ -45,8 +47,19 @@ func (a *App) SetPluginKeyWithExpiry(pluginId string, key string, value []byte,
 }
 
 func (a *App) GetPluginKey(pluginId string, key string) ([]byte, *model.AppError) {
-	result := <-a.Srv.Store.Plugin().Get(pluginId, getKeyHash(key))
+	result := <-a.Srv.Store.Plugin().Get(pluginId, key)
+
+	if result.Err == nil {
+		kv := result.Data.(*model.PluginKeyValue)
+		return kv.Value, nil
+	} else if result.Err != nil {
```

Comment:
Nit pick: You can drop the `else` clause here. If `result.Err == nil` the method returns any way.

Reply:
Done.

Reply:
Sorry for being imprecise here. https://github.com/mattermost/mattermost-server/pull/9817/commits/d5a50afc803d0ba09268ef7b5d558011c960b811 was not what i meant. Can you please take a look at @lieut-data comment?